app/tasks/ssl.php

```diff
@@ -2,22 +2,35 @@
 
 global $cli;
 
+use Appwrite\Event\Event;
 use Utopia\App;
 use Utopia\CLI\Console;
+use Utopia\Validator\Hostname;
 
 $cli
     ->task('ssl')
     ->desc('Validate server certificates')
-    ->action(function () {
-        $domain = App::getEnv('_APP_DOMAIN', '');
+    ->param('domain', App::getEnv('_APP_DOMAIN', ''), new Hostname(), 'Domain to generate certificate for. If empty, main domain will be used.', true)
```

Comment:
@Meldiron are we documenting this command anywhere? It actually quite useful.

Reply:
I created task to update these docs to include this tip: https://appwrite.io/docs/certificates

Reply:
Addressed in PR: https://github.com/appwrite/docs/pull/167
Mentioned in PR description, closing conversation.